neuralprophet/utils.py

```diff
@@ -294,16 +294,22 @@ def get_holidays_from_country(country, df=None):
     else:
         dates = df["ds"].copy(deep=True)
         years = list({x.year for x in dates})
+    #support multiple countries
+    if isinstance(country, str):
+        country = [country]
 
-    try:
-        with warnings.catch_warnings():
-            warnings.simplefilter("ignore")
-            holiday_names = getattr(hdays_part2, country)(years=years).values()
-    except AttributeError:
+    holidays = {}
+    for c in country:
```

Comment:
Would be great for readability to use a more verbose name than `c` for country, we should consider renaming the `country` parameter to `country_list` or something to make `country` free for the individual country or go with something like `single_country`

Reply:
Thanks, I went for `single_country` as I wanted to keep country to keep it consistence with other functions' naming